sdk/datafactory/Azure.ResourceManager.DataFactory/tests/Scenario/DataFactoryDatasetResourceTests.cs

```diff
@@ -400,7 +402,7 @@ public async Task Dataset_FileShare_Create()
                 {
                     FolderPath = "Root\\MyFolder",
                     FileName = "testfilename",
-                    Format = new DatasetStorageFormat() { DatasetStorageFormatType = "Acroformat" }
+                    Format = new DatasetAvroFormat()
```

Comment:
the discriminator of this new type is `AvroFormat` which is different from the previous value, therefore here we need to confirm if the swagger is making a mistake, or we are making a mistake in the test case, and re-record.